docsite/rst/playbooks_variables.rst

```diff
@@ -409,6 +409,16 @@ To work with Base64 encoded strings::
     {{ encoded | b64decode }}
     {{ decoded | b64encode }}
 
```

Comment:
oops, sorry about that, unsuccessful merge rebase on my end. Fixed now!

Also noticed the format for when a filter was introduced is added in the docs, so updated to match that. :)